packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -109,7 +114,30 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
               {originFromZero && (
                 <span className={css(classNames.zeroTick)} style={this._getStyleUsingOffsetPercent(vertical, zeroOffsetPercent)} />
               )}
-              <span ref={this._thumb} className={classNames.thumb} style={this._getStyleUsingOffsetPercent(vertical, thumbOffsetPercent)} />
+              {typeof marks === 'boolean' && marks && this._addTickmarks(css(classNames.regularTick))}
```

Comment:
The `css` helper is not needed when you have only one class to assign. It’s probably worth removing it from line 115 also or anywhere else where it’s used in the component where it only has one argument.